db/migrations/01_create_bookmarks_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE users(id SERIAL PRIMARY KEY, email VARCHAR(120) UNIQUE, password VARCHAR(120), name VARCHAR(120), username VARCHAR(120) UNIQUE);
```

Comment:
I did the same at first but don't forget to go back and change names of files to match your current code

Reply:
UNIQUE is a good method - if you added two of the same does it just reject the second?